lib/coverband/tasks.rb

```diff
@@ -20,20 +20,23 @@
             require baseline_file
           end
         end
-        if defined? Rails
-          Dir.glob("#{Rails.root}/app/**/*.rb").sort.each { |file| 
+        if defined? Rails && !Rails.config.eager_load
```

Comment:
Does this mean when `Rails.config.eager_load` is false coverband wouldn't load anything? I guess basically in that case baseline just doesn't work...